Shared/Model/SyncCredential.swift

```diff
@@ -10,6 +10,11 @@ struct SyncCredential {
     let isNew: Bool
 }
 
+let OfflineSyncCredential = SyncCredential(
+    syncInfo: SyncUnlockInfo(kid: "", fxaAccessToken: "", syncKey: "", tokenserverURL: ""),
+    isNew: false
+)
+
```

Comment:
Where are the credentials store? IIRC, the application (i.e. not the lib) is responsible for storing them, and we're doing it the key chain.

Reply:
yes, we are doing that in the `AccountStore`. We don't persist the "offline" credentials in this implementation.